.openshift-ci/clusters.py

```
@@ -138,19 +138,41 @@ def sigint_handler(self, signum, frame):
+    OPERATOR_VERIFY = "scripts/ci/ibmcloudz-verify-operators.sh"
+    OPERATOR_TIMEOUT = 25 * 60  # 25 minutes
 
     def provision(self):
         kubeconfig = os.environ["KUBECONFIG"]
 
         print(f"Using kubeconfig from {kubeconfig}")
 
+        # For IBM Cloud Z (s390x), verify cluster access with wrapper
+        if os.environ.get("REMOTE_CLUSTER_ARCH") == "s390x":
+            print("IBM Cloud Z (s390x) detected - running provisioning verification")
+            subprocess.run(
```

Ruff (0.15.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)